pkg/webhook/validation/dynakube/namespace_selector.go

```diff
@@ -33,3 +35,17 @@ func conflictingNamespaceSelector(ctx context.Context, dv *dynakubeValidator, dy
 	}
 	return ""
 }
+
+func namespaceSelectorMatchLabelsViolateLabelSpec(ctx context.Context, _ *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
+	var errs []string
+	matchLabels := dynakube.NamespaceSelector().MatchLabels
+	for _, v := range matchLabels {
+		if v != "" {
+			errs = validation.IsValidLabelValue(v)
+		}
+	}
```

Comment:
why not use `validation.ValidateLabels` ?

it needs a field path, that is only used for creating error message. (based on example I found: `field.NewPath("spec", "namespaceSelector", "matchLabels")`)